### PR TITLE
created h2 storage dataset

### PR DESCRIPTION
**Pull request by bethanyconnolly**
* The dataset linked in the github was not working for me 
* I tracked down what I think is a different version of it from the same source (U.S DOE)
* Not really sure how good this dataset is for h2 storage - it is very small and has missing values. 
* Units were missing for H2 storage columns and most papers were not accessible to me so it was hard to check, but I am guessing it is in wt% units.
* There are some other data files in the same location which we could consider getting too
* Also a copy of this agreement needs to go anywhere we make a copy of this dataset https://datahub.hymarc.org/dataset/hydrogen-storage-materials-db/resource/955ce3f9-f8fc-4167-96ce-cb9fd3141533

### CODE REVIEW

**Comment by phalem** [comment]
@bethanyconnolly until reviewer reply, can you add the original data you found to
[awesome-chemistry-datasets](https://github.com/phalem/awesome-chemistry-datasets)

**Comment by bethanyconnolly** [comment]
> awesome-chemistry-datasets

I have now updated the link to the new dataset in [awesome-chemistry-datasets](https://github.com/phalem/awesome-chemistry-datasets)

File: data/h2_storage_materials/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      units: wt%
+      type: continuous
+      names:
+          - experimental hydrogen storage capacity
+    - id: synthetic_information
+      description: brief description of synthetic procedure
+      units:
+      type: string
+      names:
+          - synthetic procedure
```
Comment (kjappelbaum) [inline data/h2_storage_materials/meta.yaml:22]: This is an interesting column. I felt it is more of an identifier (or something we ad to an identifier); what do you think?

File: data/h2_storage_materials/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    - id: chemical_formula
+      type: formula
+      description: chemical formulas
+license: No License Provided
+links:
+    - url: https://datahub.hymarc.org/dataset/hydrogen-storage-materials-db/resource/4ef1c494-366e-43a3-bed4-a3985de5c374
+      description: website with source data
+    - url: https://datahub.hymarc.org/dataset/ad580d95-e7e2-4ef4-a7f6-3b2f91a96eba/resource/4ef1c494-366e-43a3-bed4-a3985de5c374/download/hydstormatdb-reversible_hydrides.csv
+      description: original_dataset
+num_points: 30
```
Comment (kjappelbaum) [inline data/h2_storage_materials/meta.yaml:36]: That is little :D

File: data/h2_storage_materials/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      names:
+          - synthetic procedure
+identifiers:
+    - id: material_name
+      type: IUPAC
+      description: chemical name
+    - id: chemical_formula
+      type: formula
+      description: chemical formulas
+license: No License Provided
```
Comment (kjappelbaum) [inline data/h2_storage_materials/meta.yaml:30]: shall I email the maintainer? emnadmin@nrel.gov

**Review by kjappelbaum — APPROVED** [review]

**Pull request merged**